src/components/sections/connect-four/parent/ConnectFour.route.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { Route } from 'react-router';
-import { ConnectFourContainer } from './ConnectFour.container';
+import ConnectFourComponent from './ConnectFour.component';
 
 const connectFourRoute: React.ReactNode = (
-  <Route key='game-container' component={ConnectFourContainer} />
+  <Route key='connect-four' component={ConnectFourComponent} />
```

Comment:
this file's not totally necessary since it's the root component which could be rendered directly into the `app.component`